app/models/observation.rb

```diff
@@ -345,12 +345,59 @@ def owners_votes
   # Get a list of this User's Votes for this Observation.
   def users_votes(user)
     result = []
-    for n in namings
+    namings.each do |n|
       if v = n.users_vote(user)
         result << v
       end
     end
-    return result
+    result
+  end
+
+  # list of Owners's "favorite" votes for this Observation
+  def owner_favorite_votes
+    votes.where(user_id: user_id, favorite: true)
+  end
+
+  def owner_favorite_vote
+    owner_favorite_votes[0]
```

Comment:
`.first` is clearer, so I will change it to `.first`.  Indeed, when I did the 3.2 => 4.0 upgrade, I changed a bunch of `[0]`s to `.first`s.  I used `[0]` hear because I recently read that it's faster than `.first`.  See https://github.com/JuanitoFatas/fast-ruby?utm_source=rubyweekly&utm_medium=email. I guess that my personal recency bias.